tests/onnx/data/reference_graphs/quantization/MaskRCNN-12.dot

```diff
@@ -39,10 +39,10 @@ strict digraph  {
 "37 24" [id=37, type=Conv];
 "38 27" [id=38, type=Mul];
 "39 28" [id=39, type=Add];
-"40 QuantizeLinear_29_1" [id=40, type=QuantizeLinear];
-"41 DequantizeLinear_29_1" [id=41, type=DequantizeLinear];
-"42 QuantizeLinear_35_1" [id=42, type=QuantizeLinear];
-"43 DequantizeLinear_35_1" [id=43, type=DequantizeLinear];
+"40 QuantizeLinear_35_1" [id=40, type=QuantizeLinear];
```

Comment:
I've updated the sort method of quantizers. Now the order is topological, while before it was based on quantizer type and quantizer name.
Because of that, the order of quantizers in dots were changed